frontend/src/components/skeletons/featured-post-card-skeleton.tsx

```diff
@@ -2,32 +2,32 @@ import { Skeleton } from '@/components/ui/skeleton';
 
 export const FeaturedPostCardSkeleton = () => {
   return (
-    <div className="flex rounded-lg bg-light dark:bg-dark-card">
+    <div className="flex flex-col md:flex-row rounded-lg bg-light dark:bg-dark-card">
       {/* for image */}
-      <div className="w-1/3">
-        <Skeleton className="h-full w-full rounded-lg bg-slate-200 dark:bg-slate-700" />
+      <div className="w-full md:w-1/3">
+        <Skeleton className="h-48 md:h-full w-full rounded-lg bg-slate-200 dark:bg-slate-700" />
```

Comment:
here i guess the proper one should be sm:w-1/3 instead of md: w-1/3.

Pro tip just pick the changes done in the actual post cards to these skeleton, to maintain consistency, it should be in sync .